basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/AbstractCacheTestCase.java

```diff
@@ -47,6 +50,12 @@
     @Page
     public HibernateCachePage page;
 
+    @BeforeClass
+    public static void setUp() {
```

Comment:
Please rename this method to avoid it's overriddening by `@Before public void setUp()` in inherited classes, e.g. `ReplicatedCacheTestCase`